<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>801</width>
      <height>711</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="20" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>D:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>B:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>C:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>a/(b-c)*(d+e)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(a+b)*(c-d)/e</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>a-(b+c*d)/e</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>a/b-((c+d)*e)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>a*(b-c+d)/e</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(a+b)*(c-d)/e</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>a*(b-c)/(d+e)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>a/(b*(c+d))-e</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(a+(b/c-d))*e</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>a*(b+c)/(d-e)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>a-(b/c*(d+e))</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(a-b)/(c+d)*e</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>a/(b+c-d*e)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>a*(b-c)/(d+e)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(a+b*c)/(d-e)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3"/>
     </item>
     <item row="16" column="0">
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser_2"/>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>А:</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>E:</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Посчитать</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
